tests/default.nix

```
@@ -11,5 +11,5 @@ let
 in
 
 {
-  hpos-api-tests = callPackage ./hpos-api-tests {};
+  /* hpos-api-tests = callPackage ./hpos-api-tests {}; */
```

I'm surprised this comment works. I've never seen this is nix code :smile: 

```suggestion
  // hpos-api-tests = callPackage ./hpos-api-tests {};
```

I'm not sure if it matters but I'll suggest this change anyway